superset/key_value/commands/create.py

```diff
@@ -50,7 +47,5 @@ def validate(self) -> None:
         pass
 
     @abstractmethod
-    def create(
-        self, actor: User, resource_id: int, key: str, value: str
-    ) -> Optional[bool]:
+    def create(self, args: Args,) -> Optional[bool]:
```

Comment:
Bycatch - shouldn't the return type here be `bool`, not `Optional[bool]`?

Reply:
Yes